src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -27,6 +27,7 @@ static constexpr std::string_view SnapOnInputKey{ "snapOnInput" };
 static constexpr std::string_view CursorColorKey{ "cursorColor" };
 static constexpr std::string_view CursorShapeKey{ "cursorShape" };
 static constexpr std::string_view CursorHeightKey{ "cursorHeight" };
+static constexpr std::string_view DoubleClickDelimitersKey{ "doubleClickDelimiters" };

```

Comment:
let's go with `wordDelimiters` -- this can be used for other things in the future.

Reply:
Is there a _compelling reason_ it should be profile-affine?

Reply:
Yeaaaaaaah. You're right.

I also need to update the documentation for the settings.